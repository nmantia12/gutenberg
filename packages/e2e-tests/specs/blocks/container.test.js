/**
 * WordPress dependencies
 */
import {
	searchForBlock,
	getEditedPostContent,
	createNewPost,
} from '@wordpress/e2e-test-utils';

describe( 'Container', () => {
	beforeEach( async () => {
		await createNewPost();
	} );

	it( 'can be created by using the block inserter', async () => {
		await searchForBlock( 'Container' );
		await page.click( '.editor-block-list-item-container' );
		await page.keyboard.type( 'Container block' );

		expect( await getEditedPostContent() ).toMatchSnapshot();
	} );
} );
