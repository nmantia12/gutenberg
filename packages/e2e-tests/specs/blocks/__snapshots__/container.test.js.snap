// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container can be created by using the block inserter 1`] = `
"<!-- wp:container -->
<div class=\\"wp-block-container has-primary-background-color has-background\\"><!-- wp:paragraph -->
<p>Container block</p>
<!-- /wp:paragraph --></div>
<!-- /wp:container -->"
`;
